SearchEngineCS/Phase5/Search.Test/CalculatorTest.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Collections.Generic;
+using Xunit;
+using Moq;
+using SearchLibrary;
+namespace Search.Test
+{
+    public class CalculatorTest
+    {
+        InvertedIndex Inverted;
+        [Fact]
+        public void CalculateTest()
+        {
+            SetUp();
+            var cal = new Calculator(Inverted);
+
+            var mUserInput = new Mock<IUserInput>();
+            mUserInput.Setup(x => x.ScanInput()).Returns("day +spider -rainy");
+            var qp = new QueryProcessor(mUserInput.Object);
+            qp.Process();
+            var result = cal.Calculate(qp);
+            Assert.Equal(new HashSet<string>() { "72", "49" }, result);
+
+        }
+        public void SetUp()
+        {
+            var docs = new HashSet<Document>();
```

Comment:
Return `indevrtedIndex` object instead of using a field for it. (Avoid using a `void` method in such cases)